ui/js/controllers/perf/compare.js

```diff
@@ -127,6 +137,10 @@ perf.controller('CompareChooserCtrl', [
 
                 ThResultSetModel.getResultSetsFromRevision($scope.newProject.name, $scope.newRevision).then(
                     function () {
+                        localStorageService.set('originalProject', $scope.originalProject, "sessionStorage");
+                        localStorageService.set('originalRevision', $scope.originalRevision, "sessionStorage");
+                        localStorageService.set('newProject', $scope.newProject, "sessionStorage");
+                        localStorageService.set('newRevision', $scope.newRevision, "sessionStorage");
```

Comment:
I would just store the name of originalProject (they are objects with several properties) here.